docs/core-spec/core-spec.mng

```diff
@@ -169,9 +169,10 @@ A program is just a list of bindings:
 
 \gram{\ottt}
 
-\ctor{ForAllTy}s are represented in two different ways, depending on whether
-the \ctor{ForAllTy} is anonymous (written $[[t1 -> t2]]$) or
-named (written $[[forall n . t]]$).
+\ctor{FunTy} is the special case for non-dependent function type. The
+\ctor{TyBinder} in \ghcfile{types/TyCoRep.lhs} distinguishes whether a binder is
```

Comment:
But `TyBinder`s handle only non-anonymous binders. I think this needs a bit more revision.

Reply:
Why is that? I thought the `Anon` constructor of `TyBinder` deals with anonymous binders.

Reply:
Oops -- sorry. I was thinking `TyVarBinder`.